### DISCUSSION TITLE
zig git refs

### DISCUSSION THREAD

**Discussion by atk** [oven-sh/bun#2782] [Q&A] [answered]
For void linux, we need the git ref of the zig version used to compile the binaries in order to be able to reproduce the build.

Is this information available anywhere?

**Comment by Jarred-Sumner** [comment, marked as answer]
There are two places:

`process.versions.zig`
<img width="227" alt="image" src="https://user-images.githubusercontent.com/709451/236331437-46501225-ea58-4c6f-8876-bb3b4d71dc92.png">

`ZIG_VERSION`:

https://github.com/oven-sh/bun/blob/243dfc177af1bbdea70c303d3fab790fea1d5b32/Dockerfile#L15

We are planning to move to the next versioned release of zig (though if they ship the compiler performance improvement we might again switch to git SHAs to speed up our development environment)

**Reply by atk** [reply]
Thank you!
